src/mbgl/tile/tile_worker.hpp

```diff
@@ -32,6 +33,8 @@ class TileParseResultBuckets {
 public:
     TileData::State state = TileData::State::invalid;
     std::unordered_map<std::string, std::unique_ptr<Bucket>> buckets;
+    std::unique_ptr<FeatureIndex> featureIndex;
```

Comment:
The additional members make `TileParseResultBuckets` (even more of) a misnomer. Probably what we need here is a radical refactor so that parsing a vector tile _returns_ `VectorTileData`. But in the absence of that, can we rename `TileParseResultBuckets` to `TileParseSuccess` or something?